Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/Feature/SubtreeTagging.php

```diff
@@ -146,20 +149,36 @@ private function moveSubtreeTags(ContentStreamId $contentStreamId, NodeAggregate
                   ' . $this->getTableNamePrefix() . '_hierarchyrelation th
                   INNER JOIN ' . $this->getTableNamePrefix() . '_node tn ON tn.relationanchorpoint = th.childnodeanchor
                 WHERE
-                  tn.nodeaggregateid = :nodeAggregateId
+                  tn.nodeaggregateid = :newParentNodeAggregateId
                   AND th.contentstreamid = :contentStreamId
                   AND th.dimensionspacepointhash = :dimensionSpacePointHash
                 UNION
-                SELECT JSON_MERGE(cte.subtreetagsToInherit, JSON_KEYS(JSON_MERGE_PATCH(\'{}\', dh.subtreetags))) subtreeTagsToInherit, dh.childnodeanchor
+                SELECT
+                    JSON_MERGE_PRESERVE(
```

Comment:
good catch – `JSON_MERGE` and `JSON_MERGE_PRESERVE` are synonymous but the former is deprecated with MySQL 8+

Reply:
yep ^^